tests/endpoints/test_wells4run.py

```diff
@@ -0,0 +1,57 @@
+from fastapi.testclient import TestClient
+
+from tests.fixtures.well_data import load_data4well_retrieval, load_dicts_and_users
+
+
+def test_no_run_name_url(test_client: TestClient):
+
+    response = test_client.get("/pacbio/run?run_name=xxxx")
+    assert response.status_code == 404
+
+
+def test_nonexisting_run(test_client: TestClient):
+
+    response = test_client.get("/pacbio/run/xxxx")
+    assert response.status_code == 200
+    resp = response.json()
+    assert resp["page_size"] == 20
+    assert resp["page_number"] == 1
+    assert resp["total_number_of_items"] == 0
+    assert ("qc_flow_status" in resp) is False
+    assert type(resp["wells"]) is list
+    assert len(resp["wells"]) == 0
+
+
+def test_existing_run(test_client: TestClient, load_data4well_retrieval):
+
+    response = test_client.get("/pacbio/run/TRACTION_RUN_1")
```

Comment:
Are we completely happy that /pacbio/run returns lists of wells but nothing else? Is the set of wells sufficient to define the run?